README.md

```diff
@@ -1 +1,312 @@
+    "Role": "arn:aws:iam::908363916138:role/lambda-role", 
+    "Timeout": 10, 
+    "LastModified": "2019-02-02T00:28:02.658+0000", 
+    "Handler": "image-text-processor.processImages", 
+    "Runtime": "provided", 
+    "Description": ""
+}
+
+$ aws lambda update-function-configuration --function-name processImages --layers arn:aws:lambda:us-west-2:908363916138:layer:ballerina-0_990_3-runtime:19
+{
+    "Layers": [
+        {
+            "CodeSize": 175675006, 
+            "Arn": "arn:aws:lambda:us-west-2:908363916138:layer:ballerina-0_990_3-runtime:19"
+        }
+    ], 
+    "FunctionName": "processImages", 
+    "LastModified": "2019-02-02T00:30:55.803+0000", 
+    "RevisionId": "7637cae8-703b-489a-87af-ecdae62b1816", 
+    "MemorySize": 1024, 
+    "Version": "$LATEST", 
+    "Role": "arn:aws:iam::908363916138:role/lambda-role", 
+    "Timeout": 10, 
+    "Runtime": "provided", 
+    "TracingConfig": {
+        "Mode": "PassThrough"
+    }, 
+    "CodeSha256": "lwg8/90iroOzc5fFYU4Fp9r9CpMnsWgx2xOPekldHdA=", 
+    "Description": "", 
+    "CodeSize": 41462, 
+    "FunctionArn": "arn:aws:lambda:us-west-2:908363916138:function:processImages", 
+    "Handler": "image-text-processor.processImages"
+}
+```
+
+### Updating Connector Credential Environment Variables
+
+The credentials for the Amazon Rekognize and GMail connectors can be provided by setting environment variables for the target AWS Lambda function. The following command demonstrates this action.
+
+```bash
+aws lambda update-function-configuration --function-name processImages --environment "Variables={AMAZON_ACCESS_KEY=AXXXJKKK,AMAZON_SECRET_KEY=/XXXAAAA,GMAIL_CLIENTID=AAAXXXX.apps.googleusercontent.com,GMAIL_CLIENTSECRET=XXXXAAAAXXXXBBBBBBBXXXXX,GMAIL_REFRESHTOKEN=XXXXOFFXXXXX,GMAIL_ACCESSTOKEN=XXXX.XXX}"
+```
+Fill in the environment variables values with your own API credentials. The above updates can also be done using the AWS Lambda web console, by navigating to the respective function's configuration page. 
+
+Now the Ballerina AWS Lambda function `processImages` is fully deployed and configured to be used. 
+
+## Testing 
+
+We will be triggering our `processImages` function using events generated from an S3 bucket. Users will be uploading images to a specific bucket, and we will configure this bucket to send events to our lambda function when new objects are added. A guide on enabling event notifications for an S3 bucket can be found [here](https://docs.aws.amazon.com/AmazonS3/latest/user-guide/enable-event-notifications.html). In configuring this, under events, you should chose "All object create events", and for the notification destination, select "Lambda Function" and select our `processImages` as the target function. 
+
+Now, we can simply upload an image with some text to the configured S3 bucket ([example]((https://upload.wikimedia.org/wikipedia/commons/a/af/Atomist_quote_from_Democritus.png)), and you should be able to get an email similar to the following:
```

Comment:
```suggestion
Upload an image with some text to the configured S3 bucket example, and you will receive an email similar to the following:
(https://upload.wikimedia.org/wikipedia/commons/a/af/Atomist_quote_from_Democritus.png)
```

Reply:
The example is the image input given, it's not a "bucket example". I will keep the image link as it is and update rest of the sentence as suggested.